DER/RateDerivatives/IRSwaps.rdf

```diff
@@ -105,7 +106,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>cross-currency interest rate swap</rdfs:label>
-		<skos:definition>an interest rate swap in which the two streams of interest payments are in different currencies</skos:definition>
+		<skos:definition>interest rate swap in which the two streams of interest payments are in different currencies</skos:definition>
```

Comment:
please put trivial changes of wordings into their own issue - it makes it a lot more painful to review substantive important changes with all this clutter!